kratos/utilities/entities_utilities.cpp

```diff
@@ -0,0 +1,211 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/entities_utilities.h"
+
+namespace Kratos
+{
+namespace EntitiesUtilities
+{
+
+void InitializeEntities(ModelPart& rModelPart)
```

Comment:
Can't  you  all `InitializeElements` , `InitializeCondition` and `InitializeMasterSlaveConstraints` functions inside this function ? that way you can reduce reimplementing ?

Reply:
It is done in a independent PR, I am waiting fro approval #6389

Reply:
Sorry then I did not see #6389 . That addresses my questions here !